### PR TITLE
fix: drop extra .tgz from release artifact

### PR DESCRIPTION
**Pull request by jessesimpson36**
Related to #463 

This commit will remove the duplicate .tgz extension from the tarball release artifact.

### CODE REVIEW

**Comment by mgoerens** [comment]
Thanks for your contribution !

This LGTM, @komish ?

**Review by komish — CHANGES_REQUESTED** [review]
Thank you! Can you remove this modification from your commit?

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -124,4 +124,4 @@ jobs:
             latest
           registry: quay.io/redhat-certification
           username: ${{ secrets.QUAY_BOT_USERNAME }}
-          password: ${{ secrets.QUAY_BOT_TOKEN }}
+          password: ${{ secrets.QUAY_BOT_TOKEN }}
```
Comment (komish) [inline .github/workflows/release.yaml:127]: Can you remove this from your commit?

**Comment by jessesimpson36** [comment]
I'm sorry, I may have made my modifications too fast. @komish , I removed the whitespace edit I unintentionally added. I'm unclear on if there was something else in your review you would like, as you said 

> whoops - didn't highlight the code. See the next review request for a more accurate representation of what I meant

I think you only meant to highlight the added newline character.

**Comment by komish** [comment]
@jessesimpson36 You nailed it, thank you!

A quick sidenote, though, I'm fairly certain this doesn't fix the problem you're seeing, but it's still an extra tgz where it doesn't belong, so we'll get this merged shortly.

The issue sits in how openshift-tools-installer is filtering our release assets. I'll look to pursue this either in our pipelines (remaining our assets) or directly with openshift-tools-installer itself. If I pursue the latter, I'll CC you into the conversation, should you want to follow along.

For now, though, I've attached an extra release asset to our 1.13.6 so that your pipeline (and ours) will pass when using openshift-tools-installer. 

In my local testing with openshift-tools-installer, it picks up the right release asset now.

```
Download path of chart-verifier is https://github.com/redhat-certification/chart-verifier/releases
::debug::chart-verifier 1.13.6 files: chart-verifier-1.13.6-linux-amd64.tgz, chart-verifier-1.13.6-sbom.spdx.json, chart-verifier-1.13.6.tgz
Current operating system is linux
::debug::After filterByOS, 1 files remain.
::debug::1 clients remaining; skipping filterByArch.
::debug::1 clients remaining; skipping filterByZipped.
Selecting chart-verifier-1.13.6-linux-amd64.tgz
```

My apologies! We'll try to get this resolved before our next release.

**Comment by jessesimpson36** [comment]
You're the best. Thank you.

**Review by komish — APPROVED** [review]
/lgtm

**Pull request merged**